packages/flutter/lib/src/material/search_anchor.dart

```diff
@@ -407,6 +411,20 @@ class SearchAnchor extends StatefulWidget {
   /// Defaults to true.
   final bool enabled;
 
+  /// Configures how smart dashes are handled in the text field
+  /// used by this [SearchAnchor].
+  ///
+  /// For example, when enabled, double hyphens (`--`) may be
+  /// automatically replaced with an em dash (`—`) on iOS.
```

Comment:
Can you add a "See also" section that points to TextField.smartDashesType? Also, add a sentence about the default "Defaults to SmartDashesType.enabled."